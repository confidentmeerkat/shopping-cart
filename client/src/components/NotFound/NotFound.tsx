import * as React from 'react';
import '@styles/NotFound.css';

const NotFound = () => (
  <div className="not-found-container">
    <div className="not-found">
      <h1>404</h1>
      <h3>Page Not Found</h3>
    </div>
  </div>
);

export default NotFound;
