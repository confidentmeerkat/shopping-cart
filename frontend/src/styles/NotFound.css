.not-found-container {
  min-height: 100vh;
  margin-top: -72px;
  border: 1px solid #ffffff00;
}

.not-found {
  width: 50%;
  min-width: 330px;
  margin: auto;
  margin-top: 200px;
}

.not-found h1 {
  font-size: 60px;
  color: #00BCD4;
}

.not-found h3 {
  font-size: 45px;
}