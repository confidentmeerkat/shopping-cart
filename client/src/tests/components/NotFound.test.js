import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../components/NotFound';

describe('<NotFound />', () => {
  it('renders NotFound correctly', () => {
    const wrapper = shallow(<NotFound />);

    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('h1').text()).toBe('404');
    expect(wrapper.find('h3').text()).toBe('Page Not Found');
  });
});