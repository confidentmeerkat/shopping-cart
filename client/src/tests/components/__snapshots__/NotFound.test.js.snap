// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> renders NotFound correctly 1`] = `
<div
  className="not-found-container"
>
  <div
    className="not-found"
  >
    <h1>
      404
    </h1>
    <h3>
      Page Not Found
    </h3>
  </div>
</div>
`;
